odoo_sh/getting_started/branches.rst

```diff
@@ -0,0 +1,310 @@
+
+The branches view gives you an overview of the different branches your repository has.
+
+.. image:: ./media/interface-branches.png
+   :align: center
+
+Stages
+===============
+
+Odoo.sh offers three different stages for your branches: production, staging and development.
+
+You can change the stage of a branch by drag and dropping it on the stage section title.
+
+.. image:: ./media/interface-branches-stagechange.png
+   :align: center
+
+Production
+----------
+This is the branch holding the code on which your production database run.
+There can be only one production branch.
+
+When you push a new commit in this branch,
+your production server is updated with the code of the new revision and is then restarted.
+
+If your changes require the update of a module, such as for a change in a form view,
+and you want it to be performed automatically,
+increase the version number of the module in its manifest (*__manifest__.py*).
+The platform will then take care to perform the update.
+
+In the case the changes in the commit prevent the server to restart,
+or the modules update fails,
+the server is automatically reverted to the previous successful code revision and
+the database is roll-backed as it was before the update.
+You nevertheless have access to the logs of the failed updates so you can analyze what happened.
+
+The demo data is not loaded, as it is not meant to be used in a production database.
+The unit tests are not performed, as it would increase the unavailabity time of the production database during the updates.
+
+Staging
+-------
+Staging branches are meant to test your new features using the production data.
+
+When you push a new commit in one of these branches,
+a new server is started, using a duplicate of the production database and the new revision of the branch.
+
+You can therefore test your latest features using the production data without compromising the actual
+production database with test records.
+
+The outgoing emails are not sent: They are intercepted by the mailcatcher,
+which provides an interface to preview the emails sent by your database.
+That way, you do not have to worry about sending test emails to your contacts.
+
+Scheduled actions are disabled. If you want to test them, you have to enable them or trigger their action manually.
+Be careful though: If these actions perform changes on third-party services (FTP servers, email servers, ...)
+used by your production database as well,
+you might make unwanted changes in your production.
+
+The databases created for staging branches are meant to live at least two weeks.
+After that, they can be garbage collected automatically.
+If you make configuration changes or view changes in these branches, make sure to document them or write them directly
+in the modules of the branch, using XML data files overriding the default configuration or views.
+
+The unit tests are not performed as, in Odoo, they currently relies on the demo data, which are not loaded in the
```

Comment:
Better = "The unit tests are not performed as, in Odoo, they currently rely on the demo data, which is not loaded in the"